scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -137,7 +147,13 @@ RUN set -ex; \
     cp abicoder.hpp /usr/include; \
     rm -rf /usr/src/Yul-Isabelle
 
+# Cleanup
+RUN set -ex; \
+	rm -rf /var/lib/apt/lists/*
+
```

Comment:
This cleanup is actually both superfluous and ineffective.

Thanks to the `FROM base` bit below we discard everything except for a few selected dirs anyway so we don't need to worry about cleaning up anywhere outside of these dirs.

 And if we weren't doing that, removing stuff in a separate `RUN` command would not really remove it from the image. It would still be in the original layer and just marked as removed by layer above, but still taking up space.

Reply:
Oh, right. Didn't know that. We can remove those then.